src/Transport/Config/AzureServiceBusTransport.cs

```diff
@@ -17,10 +18,17 @@ protected override TransportInfrastructure Initialize(SettingsHolder settings, s
 
             RegisterConnectionStringAsNamespace(connectionString, settings);
 
+            SetConnectivityMode(settings);
```

Comment:
Good question @melkio! It was initially, but I've realized that one can't see the forest for the trees. Similar to `RegisterConnectionStringAsNamespace` that could be done as 
```c#
var namespaces = settings.Get<NamespaceConfigurations>(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces);
              namespaces.AddDefault(connectionString).AddDefault(connectionString)
```
but the readability of that is horrendous. I.e. "tell me what it does, not how" approach :)